src/components/TodoForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { useTodoContext } from '../context/TodoContext';
+import { addTodo } from '../api/todos';
+import { Error } from '../types/Error';
```

Comment:
It's better to use custom error handling instead of using a generic Error type. Create a custom Error enum in the 'types' folder and import it here.